pkg/cmd/repo/create/create.go

```diff
@@ -164,11 +173,45 @@ func createRun(opts *CreateOptions) error {
 		}
 	}
 
+	// find template ID
+
+	if opts.Template != "" {
+		httpClient, err := opts.HttpClient()
+		if err != nil {
+			return err
+		}
+
+		var toView ghrepo.Interface
+		apiClient := api.NewClientFromHTTP(httpClient)
+
+		// var err errors
+		viewURL := opts.Template
+		if !strings.Contains(viewURL, "/") {
+			currentUser, err := api.CurrentLoginName(apiClient, ghinstance.Default())
+			if err != nil {
+				return err
+			}
+			viewURL = currentUser + "/" + viewURL
+		}
+		toView, err = ghrepo.FromFullName(viewURL)
+		if err != nil {
+			return fmt.Errorf("argument error: %w", err)
+		}
+
+		repo, err := api.GitHubRepo(apiClient, toView)
```

Comment:
the `toView` naming doesn't make much sense in this context; something like `toClone` would read better.

Reply:
Addressed in https://github.com/cli/cli/pull/1590/commits/2886dd913f1621646b960fd61d0f1672bba9b64d, thanks for the reminder!